sycl/source/detail/graph_impl.hpp

```diff
@@ -989,6 +1050,8 @@ class exec_graph_impl {
   std::vector<sycl::detail::AccessorImplPtr> MAccessors;
   /// List of all execution events returned from command buffer enqueue calls.
   std::vector<sycl::detail::EventImplPtr> MExecutionEvents;
+  /// List of the partitions that compose the exec graph
```

Comment:
About as nitpicky as you can get
```suggestion
  /// List of the partitions that compose the exec graph.
```